region/src/main/java/org/motechproject/nms/region/utils/GetBoolean.java

```diff
@@ -0,0 +1,26 @@
+package org.motechproject.nms.region.utils;
+
+import org.motechproject.nms.region.exception.CsvImportDataException;
+import org.supercsv.cellprocessor.ift.CellProcessor;
+import org.supercsv.util.CsvContext;
+
+public class GetBoolean implements CellProcessor {
+
+    @Override
+    public Object execute(Object value, CsvContext context) {
+        Boolean returnValue;
+        if (value instanceof Boolean) {
+            returnValue = (Boolean) value;
+        } else if (value instanceof String) {
+            returnValue = Boolean.valueOf((String) value);
```

Comment:
The sample file they sent uses 'Y' and 'N' for boolean values.  (I added the file to the ticket)